### ISSUE TITLE
Youtube, Google Play music, Spotify,.. etc links in message can be open only in Safari or other browser.

### ISSUE THREAD

**Issue by finesky** [peter-iakovlev/Telegram#152]
Hello,

Is there possibility to open Youtube, Spotify and Google Play music apps when pressing corresponding link? Right now everything opens in Safari. 

Thanks
Sasha

**Comment by max-pub** [comment]
I agree, please add an option to disable the in-app-browser, so that links will open in the corresponding apps as they should
